src/app/dashboard/Missions/MissionEditView.jsx

```diff
@@ -0,0 +1,368 @@
+  },
+  foodBoxDetailName: {
+    padding: theme.spacing(1),
+  },
+}));
+
+/**=====BASE COMPONENTs======**/
+
+const Label = ({ children, classes }) => {
+  if (!children) return null;
+  return (
+    <Body2 align="left" className={classes.rowLabel} color="textPrimary">
+      <b>{children}</b>
+    </Body2>
+  );
+};
+
+const Row = ({ children, classes, Icon }) => {
+  if (!children) return null;
+  return (
+    <Grid container className={classes.rowBody}>
+      <Box marginRight="20px" width="20px">
+        {Icon && <Icon color="primary" />}
+      </Box>
+      {children}
+    </Grid>
+  );
+};
+
+const Card = ({ children, classes, label }) => {
+  if (!children) return null;
+  return (
+    <>
+      <Label classes={classes}>{label}</Label>
+      {children}
+    </>
+  );
+};
+
+/**=====ROW COMPONENTS=======*/
+
+const MissionImage = ({ classes, mission }) => {
+  const imageUrl = mission?.image;
+  if (!imageUrl) return null;
+
+  return (
+    <Container>
+      <img src={imageUrl} className={classes.missionImage} alt="details" />
+    </Container>
+  );
+};
+const MissionTypeRow = ({ classes, mission }) => {
+  let missionType;
+  switch (mission?.type) {
+    case "foodbox":
+    default:
+      missionType = "Food Box";
+  }
+  return (
+    <Box marginTop="32px">
+      <H3>{missionType}</H3>
+    </Box>
+  );
+};
+const MissionStatusRow = ({ classes, mission }) => {
+  let status = mission?.status;
+  let missionStatusText;
+  switch (status) {
+    case Mission.Status.unassigned:
+      missionStatusText = "Looking for volunteer";
+      break;
+    default:
+      missionStatusText = status;
+      break;
+  }
+  return (
+    <Row Icon={PanToolIcon} classes={classes}>
+      {missionStatusText}
+    </Row>
+  );
+};
+
+const MissionFundedStatusRow = ({ classes, mission }) => {
+  let missionFundedStatusText;
+  switch (mission?.fundedStatus) {
+    case Mission.FundedStatus.fundedbydonation:
+      missionFundedStatusText = "Funded By Donation";
+      break;
+    case Mission.FundedStatus.fundedbyrecipient:
+      missionFundedStatusText = "Funded By Recipient";
+      break;
+    case Mission.FundedStatus.notfunded:
+    default:
+      missionFundedStatusText = "Not Yet Funded";
+  }
+  return (
+    <Row Icon={AttachMoneyIcon} classes={classes}>
+      {missionFundedStatusText}
+    </Row>
+  );
+};
+
+const FoodBoxDetailsRow = ({ classes, details }) => {
+  return (
+    <Box>
+      {details?.needs?.map((box, index) => (
+        <Grid key={index} container className={classes.foodBoxDetailContainer}>
+          <Grid className={classes.foodBoxDetailQuantity}>
+            <b>{box?.quantity}</b>
+          </Grid>
+          <Grid className={classes.foodBoxDetailName}>
+            <b>{box?.name}</b>
+          </Grid>
+        </Grid>
+      ))}
+    </Box>
+  );
+};
+
+const MissionDetailsRow = ({ classes, mission }) => {
+  let type = mission?.type;
+  let details = mission?.missionDetails;
```

Comment:
if you were seeing anything weird with missionDetails not showing up the DB got changed to use just `details`. 
I fixed / reflected this in the schema here https://github.com/factn/resilience-app/pull/551 so hopefully that goes in soon.

Reply:
Thanks.  Changed to ".details"